helm-charts/code-annotation/values.yaml

```diff
@@ -10,6 +10,8 @@ image:
   repository: docker.io/srcd/code-annotation
   # tag must be received as a parameter
   pullPolicy: IfNotPresent
+deployment:
+    internalDatabasePath: /var/code-annotation
```

Comment:
If there's no env variable that will inform that this may change beyond the default, I would not give the possibility of overriding it and I would hardcode it in the yaml

Reply:
I think this is what you're looking for... isn't it?
https://github.com/src-d/code-annotation/pull/90/commits/bfa612c1f6fa632f0ca3ef2c39f158d4c0d8bf12